### PR TITLE
CVE updated

### PR DESCRIPTION
**Pull request by oxelson**
This change is [Reviewable](https://reviewable.io/reviews/Unidata/tds/369)

### CODE REVIEW

File: tds-platform/build.gradle [hunk 1]
```diff
@@ -80,22 +80,22 @@ dependencies {
...
+    api 'org.glassfish:jakarta.el:4.0.2'
 
     // waterml stuff
     api 'org.apache.xmlbeans:xmlbeans'
     api 'org.n52.sensorweb:52n-xml-waterML-v20'
     api 'org.n52.sensorweb:52n-xml-om-v20'
 
     // edal-java (ncwms)
-    def edalVersion = '1.5.0.6-SNAPSHOT'
+    def edalVersion = '1.5.2'
```
Comment (haileyajohnson) [inline tds-platform/build.gradle:98]: we don't was to use their version of edal-java, we need to keep using ours

**Review by haileyajohnson — CHANGES_REQUESTED** [review]

File: tds-platform/build.gradle (hunk 1 above)
Comment (oxelson) [reply tds-platform/build.gradle:98]: Yep! Fixing that now.

File: project-files/owasp-dependency-check/dependency-check-suppression.xml [hunk 2]
```diff
@@ -110,4 +111,50 @@
...
+  <suppress>
+    <notes><![CDATA[
+      file name: screenshot_sync
+      reason: False positive.  We do not use the fredsmith utils.
+    ]]></notes>
+    <cve>CVE-2021-4277</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+      file name: je-4.0.92.jar
```
Comment (tdrwenski) [inline project-files/owasp-dependency-check/dependency-check-suppression.xml:139]: hopefully this suppression can go now that this is removed from edal-java

Comment (oxelson) [reply project-files/owasp-dependency-check/dependency-check-suppression.xml]: removed!

**Pull request merged**